### PR TITLE
Feat/baked wc rotaion data

### PR DESCRIPTION
**Pull request by sergeyWh1te**

### CODE REVIEW

File: cmd/dc4bc_cli/main.go [hunk 1]
```diff
@@ -237,6 +238,7 @@ func getBatchesRequest(host string, dkgID string) (*BatchesResponse, error) {
 	}
 
 	var response BatchesResponse
+	fmt.Println(string(responseBody))
```
Comment (sergeyWh1te) [inline cmd/dc4bc_cli/main.go:241]: @swelf19 
Mistyped?

**Review by pr0n00gler — COMMENTED** [review]
I suggest to run: 
`gofumpt -w . && goimports -w -local github.com/lidofinance/dc4bc .`
to beatify the code a bit

File: pkg/wc_rotation/rotation.go [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+	// ToExecutionAddress 0xb9d7934878b5fb9610b3fe8a5e441e8fad7e293f
+	//
+	// https://mainnet.lido.fi/#/lido-dao/0x2e59a20f205bb85a89c53f1936454680651e618e/vote/78/
+	ToExecutionAddress = [20]byte{185, 215, 147, 72, 120, 181, 251, 150, 16, 179, 254, 138, 94, 68, 30, 143, 173, 126, 41, 63}
+
+	//go:embed payloads.csv
+	ValidatorsIndexes string
+)
+
+func GetValidatorsIndexes(start, end int) ([]uint64, error) {
```
Comment (pr0n00gler) [inline pkg/wc_rotation/rotation.go:47]: Unused function

Comment (sergeyWh1te) [reply pkg/wc_rotation/rotation.go:47]: @swelf19 Do we need it?

**Comment by sergeyWh1te** [comment]
> I suggest to run: `gofumpt -w . && goimports -w -local github.com/lidofinance/dc4bc .` to beatify the code a bit

done

**Review by pr0n00gler — APPROVED** [review]

**Pull request merged**